Plugins~/Src/MeshSyncClientBlender/msblenBinder.h

```diff
@@ -115,6 +114,13 @@ namespace blender
         inline uint32_t GetNumUVs() const;
 
         void calc_normals_split();
+        void update();
+        void clear_geometry();
+        void add_vertices(int count);
+        void add_polygons(int count);
+        void add_loops(int count);
+        void add_edges(int count);
+        void add_normals(int count);
```

Comment:
is this adding empty data?

Reply:
Yeah, the python binding just allows you to make the array bigger and populate it with data afterwards.